readme.md

```diff
@@ -8,28 +8,46 @@
 
 ## Overview
 
+Chief-of-state (COS) is a dockerized persistence framework for building modern, clustered, reactive applications.
+COS encourages best practices in CQRS and event-sourcing through simple, language-agnostic interfaces via gRPC, and it
```

Comment:
I think this would work better to sell: 

COS ~encourages~ supports ~best practices in~ CQRS and event-sourcing through simple, language-agnostic interfaces via gRPC, and it